README.md

```diff
@@ -1,2 +1,64 @@
-# harry-potter-quarkus
-Showcase in a web application Quarkus and Infinispan
+# Quarkus demo: Infinispan Client
+
+This example showcases how to use Infinispan client with Quarkus.
+* Create stores dynamically
+* Simple get/put operations
+* Full-text query
+* Continuous query
+
+Uses quarkus features, such as
+* REST web-service
+* Web Socket
+* Task Scheduling
+* Application properties
+* Application lifecycle events listeners
+
+# Run infinispan server
+
+- Running with docker `docker run -it -p 11222:11222 jboss/infinispan-server:latest`
```

Comment:
Should mention you only have to do one or the other.

Reply:
ok